src/monocle_apptrace/constants.py

```diff
@@ -0,0 +1,7 @@
+AZURE_ML_ENDPOINT_ENV_NAME = "AZUREML_ENTRY_SCRIPT"
+AZURE_FUNCTION_WORKER_ENV_NAME = "FUNCTIONS_WORKER_RUNTIME"
+AZURE_APP_SERVICE_ENV_NAME = "WEBSITE_SITE_NAME"
+
+AZURE_FUNCTION_NAME = "azure_function"
+AZURE_APP_SERVICE_NAME = "azure_app_service"
+AZURE_ML_SERVICE_NAME = "azure_ml"
```

Comment:
should the naming convention be we follow azure_<Abbreviation> or azure.<abbreviation> as mentioned in the following link 
https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations ??

Reply:
If these become component types then component types use dot as instead of underscore

Current component types in API for eg:

serve.endpoint
inference.triton
inference.triton.model
retrieval.vectordb
infra.kubernetes
workflow.langchain
workflow.llamaindex
workflow.haystack
workflow.generic
azure
inference.openai
inference.azopenai
inference.openai.model
inference.azopenai.model
infra.azureml
infra.sagemaker
infra.vertex
infra.generic